src/frontend/packages/lib_components/src/common/Text/TextAreaInput/index.tsx

```diff
@@ -23,6 +23,9 @@ export const TextAreaInput = ({
         spellCheck={false}
         title={title}
         value={value}
+        style={{
+          minHeight: '150px',
+        }}
```

Comment:
```suggestion
        onInput={(e) => {
            e.currentTarget.style.height = 'auto';
            e.currentTarget.style.height = `${e.currentTarget.scrollHeight}px`;
        }}
```

[Marsha - responsive teaxtarea.webm](https://user-images.githubusercontent.com/25994652/234520127-177ac4c3-aaae-46f9-9a45-2186c3f5403e.webm)

Reply:
When we leave the page and we come back, the textarea is "collapse" and when I modifiy it then its height is resize. This is a little bit disturbing